Backend/knexfile.js

```diff
@@ -0,0 +1,49 @@
+// Update with your config settings.
+module.exports = {
+  development: {
+    // connection: process.env.DATABASE_URL,
```

Comment:
Why commented out?

Reply:
used for testing interaction with db, will use if for future test